MultiMarginCriterion.lua

```diff
@@ -1,9 +1,10 @@
 local THNN = require 'nn.THNN'
 local MultiMarginCriterion, parent = torch.class('nn.MultiMarginCriterion', 'nn.Criterion')
 
-function MultiMarginCriterion:__init(p, weights)
+function MultiMarginCriterion:__init(p, margin, weights)
```

Comment:
please make margin to be the 3rd argument. the calls need to be backward-compatible.